3ds Max/Max2Babylon/2015/Max2Babylon2015.csproj

```diff
@@ -43,7 +43,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
     <DebugSymbols>true</DebugSymbols>
-    <OutputPath>C:\Program Files\Autodesk\3ds Max 2016\bin\assemblies\</OutputPath>
+    <OutputPath>..\assemblies\2015\</OutputPath>
```

Comment:
Do we need to place all the binaries in ..\Assemblies\<version>? Why not just change the PostBuild script to retrieve the binaries from their respective bin/<build_flavor> directory?

Reply:
Sorry to hold up the PR, I just noticed this when testing the installer .bat

Reply:
I'm not sure if this is what you mean.
I used the default bin/ConfigurationName folder, then the post-build look for the right folder.

Reply:
I'm alright with using the default build location in bin/<build flavor>, or we can pass $(OutDir) as a command line parameter to the postbuild script.